frontend/src/components/ListingViewer/ListingViewer.jsx

```diff
@@ -86,219 +75,196 @@ class ListingViewer extends Component {
   }
 
   render() {
+    const listingDetail = this.props.selectedListing
     return (
-      <div className = "listingContent">
-        <div>
-          {this.props.listingArray && this.props.listingArray.length !== 0 
-          ? 
-            this.props.listingArray.slice(1,2).map((listingDetail, index)=>{ // slice and map unnecessary
-              const currIndex = this.props.pageNum * 10 + index;
-              return (
-                <Paper
-                  key = {currIndex}
-                  elevation = {3}
-                  style = {{
-                    width: "100%",
-                    marginTop: 16,
-                    marginBottom: 16
-                  }}
-                >
-                  {/* This is for the image area 
-                  
-                    The image section is going to eventually need to handle multiple images. 
-                    Maybe a for loop similar this could be a good replacement:
-                    {imagesDummyInputs.map((source, index) => {
-                          return (
-                          <img
-                              style = {{width: "30%",height: "30%", display: 'inline', marginLeft:'5px'}}
-                              src={source}
-                              alt="apartment"
-                          />
-                          );
-                      })}
-                  */}
-                  <div style={{width: '100%', marginLeft:'5px', marginTop:'5px'}}>
-                    {this.checkImageValid(listingDetail.images[0])
-                      ?
-                        <img
-                          style = {{
-                            width: "30%",
-                            height: "30%"
-                          }}
-                          src={listingDetail.images[0]}
-                          alt="apartment"
-                        />
-                      :
-                        <ApartmentIcon
-                          style = {{
-                            width: "30%",
-                            height: "30%",
-                            color: "lightgray"
-                          }}
-                          alt="apartment"
-                        />
-                    }
-                  </div>
+      <Paper
+        elevation = {3}
+        style = {{
+          width: "100%",
+          marginLeft:16
+        }}
+      > 
+        <div className = "listingDetailHeader">
+          <Typography
+            style = {{
+              fontWeight: 700,
+              fontSize: 26,
+              fontFamily: "cursive",
```

Comment:
For me this results in a Comic Sans font.

Reply:
Yeah, thought thats kind of cool, lol, ez change back to normal font